lib/PasswordStrength/PasswordStrength.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import taiPasswordStrength from 'tai-password-strength';
+import { withStripes } from '@folio/stripes-core/src/StripesContext';
+
+class PasswordStrength extends React.Component {
+  static propTypes = {
+    dataSource: PropTypes.string.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const password = this.props.dataSource;
+    if (!password) {
+      return false;
+    }
+
+    const strengthTester = new taiPasswordStrength.PasswordStrength();
+    strengthTester.addCommonPasswords(taiPasswordStrength.commonPasswords);
+    strengthTester.addTrigraphMap(taiPasswordStrength.trigraphs);
+    const result = strengthTester.check(password);
+    let output = 'weak';
+
+    switch (true) {
+      case result.strengthCode.indexOf('WEAK') >= 0:
+        output = 'weak';
+        break;
+      case result.strengthCode.indexOf('STRONG') >= 0:
+        output = 'strong';
+        break;
+      default:
+        output = 'reasonable';
+        break;
+    }
+
+    return (
+      <div>
+        <FormattedMessage id={'stripes-smart-components.password-strength.' + output}/>
+      </div>
+    );
+  }
+}
+
+export default withStripes(PasswordStrength);
```

Comment:
It doesn't look like you're accessing anything on the `stripes` god object, so I don't think you need `withStripes`. You can just `export default PasswordStrength;`.

Reply:
@cherewaty  thanks for your comment. 
I'm new to stripes platform. And I did not find a way to pass stripes to this component other then <code>withStripes</code> (I found it in another module). If you will describe how to pass <code>stripes</code> to the component, I will really appreciate.

Reply:
I think @cherewaty was suggesting that since you don't use `stripes` anywhere in this class, you don't need to wrap it with `withStripes`.

Reply:
It is needed for translation strings.

Reply:
Updated, please take a look. No more withStripes